### DISCUSSION TITLE
포스트 목록에서 링크 영역 확대 건의

### DISCUSSION THREAD

**Discussion by julyrunt** [pronist/hELLO#194] [Ideas]
현재 블로그 포스트 목록에서 해당 포스트로 이동하기 위해선 반드시 타이틀 문구를 클릭해야 합니다.

썸네일이나 설명을 포함한 다른 영역에 대해서는 링크가 걸려있지 않기 때문에 무심코 클릭하고 당황하게 됩니다.

포스트 목록에서의 링크 영역이 썸네일, 타이틀, 설명을 포함하는 포스트 블록 전체로 확대되었으면 좋겠습니다.

**Comment by pronist** [comment]
@julyrunt 안녕하세요.

포스트 블록 전체로 확대하기에는 좀 그렇고, 섬네일 쪽에 링크를 추가하는 것을 고려하고 있습니다. :)
